ci/coq-tests.el

```diff
@@ -200,6 +200,8 @@ For example, COMMENT could be (*test-definition*)"
 
 
 (ert-deftest 020_coq-test-definition ()
+  ;; There are no infomsgr when running silent.
+  :expected-result :failed
```

Comment:
We should probably test this when only one command is sent at a time.

Reply:
AFAICT there is only one command processed in this test. Even if you process one definition alone, there is no infomsg when running silent.

Reply:
You are right. this behaviour is ok for me. I wonder if someone really care about this fonfirmation messages outside pure textual coqtop use.